ang/civicase/contact-case-tab/directives/contact-case-tab.directive.html

```diff
@@ -11,7 +11,7 @@ <h1 crm-page-title class="element-invisible">Related cases to the user</h1>
     </a>
     <a ng-if="checkPerm('add cases') && newCaseWebformUrl"
        class="btn-primary btn"
-       ng-href="{{ newCaseWebformUrl | civicaseCrmUrl:{cid: contactId} }}">
+       ng-href="{{ newCaseWebformUrl | civicaseCrmUrl:{[newCaseWebformClient]: contactId} }}">
```

Comment:
Trying to wrap my head around this as it's not very clear from PR description.
What are the possible values of newCaseWebformClient ? and why are we wrapping it in `[]`?
Perhaps a bit more description in the PR will help.

Reply:
Added technical details. Pls check